### DISCUSSION TITLE
TypeError: class `EncoderDecoder` in mmseg/models/segmentors/encoder_decoder.py: class `MaskFormerHead` in mmseg/models/decode_heads/maskformer_head.py: __init__() got an unexpected keyword argument 'in_channels'

### DISCUSSION THREAD

**Discussion by zub12** [open-mmlab/mmsegmentation#3075] [General]
TypeError: class `EncoderDecoder` in mmseg/models/segmentors/encoder_decoder.py: class `MaskFormerHead` in mmseg/models/decode_heads/maskformer_head.py: __init__() got an unexpected keyword argument 'in_channels'

**Comment by HePengguang** [comment]
same issue

**Comment by HePengguang** [comment]
did you solve it?

**Comment by BeingDog** [comment]
same issue  https://github.com/open-mmlab/mmsegmentation/issues/2914

**Comment by xiexinch** [comment]
It requires installing mmdetection.
Might follow our document https://github.com/open-mmlab/mmsegmentation/tree/main/configs/mask2former#usage to use it.
